charts/pomerium/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: pomerium
-version: 13.1.3
+version: 13.1.4
```

Comment:
Please bump minor here.  I think that best adheres to semver.

Reply:
Done